internal/pkg/api/server/templates/service/service.go

```diff
@@ -87,15 +93,26 @@ func (s *service) ValidateInputs(d dependencies.Container, payload *ValidateInpu
 	if err != nil {
 		return nil, err
 	}
-	return validateInputs(tmpl.Inputs(), payload.Steps)
+
+	// Process inputs
+	result, _, err := validateInputs(tmpl.Inputs(), payload.Steps)
+	return result, err
 }
 
 func (s *service) UseTemplateVersion(d dependencies.Container, payload *UseTemplateVersionPayload) (res *UseTemplateResult, err error) {
+	// Note:
+	//   A very strange code follows.
```

Comment:
😆  ok. To vlastně udělá init pull do memory FS, aplikaci šablony a potom push? To mi nepřijde až tak strange. 👍

Reply:
Mne dost, .... aby som nastavil filter, tak som ho musel nastavit do manifestu ... take :spaghetti:

Reply:
Jo no tak provedení není úplně ideální ale vzhledem k okolnostem. 😃